.dockerignore

```diff
@@ -0,0 +1,9 @@
+.git
+.gitignore
+
+.DS_Store
+.vscode/
+coverage/
+
+.gem
+Gemfile.lock
```

Comment:
Should we be excluding the .lock file from the docker image? Wouldn't we want the version we're testing inside the docker container to use the same dependencies from the Gemfile.lock in this repo.

Reply:
Currently, the Gemfile.lock is in the .gitignore. This was advised by Bundler in the past, but is not recommended anymore. So I also added it to the .dockerignore. Otherwise someone would have to install Bundler and Ruby after all before makiing the Docker image.

I tried taking the Gemlock.file out of the .gitignore, but then the test matrix of different Rubies in GH Actions CI didn't work. I wondered about creating a temporary .gitignore - just to exclude the Gemlock.file - as part of the test job. What do you think?